contrib/pg_tde/src/common/pg_tde_utils.c

```diff
@@ -33,12 +33,18 @@ PG_FUNCTION_INFO_V1(pg_tde_is_encrypted);
 Datum
 pg_tde_is_encrypted(PG_FUNCTION_ARGS)
 {
-	Oid			tableOid = PG_GETARG_OID(0);
+	Oid			relationOid = PG_GETARG_OID(0);
 	LOCKMODE	lockmode = AccessShareLock;
-	Relation	rel = relation_open(tableOid, lockmode);
+	Relation	rel = relation_open(relationOid, lockmode);
 	RelFileLocatorBackend rlocator = {.locator = rel->rd_locator,.backend = rel->rd_backend};
 	InternalKey *key;
 
+	if (!RELKIND_HAS_STORAGE(rel->rd_rel->relkind))
```

Comment:
We should mention this in the documentation too.

Reply:
Will add that.